### DISCUSSION TITLE
exe version of VVenC

### DISCUSSION THREAD

**Discussion by berbunda** [fraunhoferhhi/vvenc#80] [Q&A] [answered]
Hi.
I am sorry for my question.
Can You make an exe-version of VVenC?
I never worked in Visual Studio and never installed it.
Please, if it is not hard for You.

**Comment by adamjw24** [comment, marked as answer]
1. Don't be sorry to ask questions.
2. We are not planning to be releasing binary version of the SW.
3. If you are familiar with GitHub, you could fork a project, add a deploy step to the CI/CD (.github/workflows) and download the executable for your fork. We would not accept this as a pull-request, because of 2., but your problem would be fixed.

Sorry I couldn't be of more help.

**Reply by berbunda** [reply]
I was able to make binary veresion of vvenc and vvdec. I am afraid to start fork project because I have never have experience with coding, programming etc.
